docs-en/chapter_data_structure/classification_of_data_structure.md

```diff
@@ -20,23 +21,28 @@ Non-linear data structures can be further divided into tree structures and netwo
 
 ## Physical Structure: Contiguous and Dispersed
 
-**When an algorithm program runs, the data being processed is mainly stored in memory**. The following figure shows a computer memory stick, each black block containing a memory space. We can imagine memory as a huge Excel spreadsheet, where each cell can store a certain amount of data.
 
-**The system accesses data at the target location through memory addresses**. As shown in the figure below, the computer allocates numbers to each cell in the table according to specific rules, ensuring each memory space has a unique memory address. With these addresses, programs can access data in memory.
+**During the execution of an algorithm, the data being processed is stored in memory**. The figure below shows a computer memory stick where each black square represents a memory space. We can imagine memory as a vast Excel spreadsheet in which each cell can store data of a specific size.
```

Comment:
represents a memory space -> is a physical memory space

Each chiplet on the memory stick ***is*** a block of the physical memory, so using "is" over "represents" is a bit more accurate.